src/route/dataset-route.ts

```diff
@@ -101,6 +113,11 @@ apiRoute.get('/:dataset', async (req, res) => {
         res.json({ message: 'Dataset not found... file is null or undefined' });
         return;
     }
+    if (isValidUUID(datasetID) === false) {
+        res.status(404);
+        res.json({ message: 'Dataset not found...File ID is not Valid.' });
```

Comment:
Is this really a fileID? We shouldn't expose filenames directly to users, even if we do check them for form. We should only expose the IDs of database objects.
Should this say something like "Malformed dataset id"?

Reply:
Changed to `Dataset ID is not valid`